jbmc/src/java_bytecode/library/CMakeLists.txt

```diff
@@ -1,2 +0,0 @@
-# create a target for the executable performing the .jar -> .inc conversion
-add_executable(java-converter converter.cpp)
```

Comment:
Well, I'm certainly not complaining if we're removing code that's no longer relevant. Maybe anyone more familiar with jbmc might want to chime in if that's actually the case?

Reply:
@peterschrammel perhaps?

Reply:
That code is not used anymore and it's unlikely to be used anytime soon.